pkg/assembler/assembler.go

```diff
@@ -493,6 +520,29 @@ func (i IngestPredicates) GetVulnerabilities(ctx context.Context) []*generated.V
 	return vulns
 }
 
+func (i IngestPredicates) GetLicenses(ctx context.Context) []generated.LicenseInputSpec {
```

Comment:
can you add a quick unit test for this similar to the others

Reply:
Will do, missed those

Reply:
done